dask/async.py

```diff
@@ -429,6 +417,10 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
     """
     assert queue
 
+    if callbacks is None:
+        callbacks = _globals['callbacks']
+    callbacks = setup_callbacks(callbacks)
```

Comment:
My normal approach here would be to normalize `callbacks` to a list

```python
if callbacks is None:
    callbacks = []
if isinstance(callbacks, Callback):
    callbacks = [callbacks]
```

and then later we iterate through the list in all cases

```python
for cb in callbacks:
    cb.start(dsk, state)
...
for cb in callbacks
    cb.pretask(key, dsk, state)
```
```